start.sh

```diff
@@ -3,6 +3,8 @@
 # Chown volumes so that nginx can read/write
 chown www-data:www-data /usr/share/nginx/html/images
 chown www-data:www-data /usr/share/nginx/html/LocalSettings.php
+chown -R www-data:www-data /tmp/extensions/
+cp -r /tmp/extensions/* /usr/share/nginx/html/extensions/
```

Comment:
Ah, good point. So I'm thinking why not just add a volume path to `/usr/share/nginx/html/extensions/docker` and then add a `VOLUME` instruction to the Dockerfile? If we do that, you would be able to load the extension in LocalSettings by just adding one more directory such as `require_once($IP/extensions/docker/MobileFrontend)`. I'm not a big fan of doing a copy at boot because then you'd lose the volume that's at `/tmp/extensions`, such that anything that you add to the extensions directory will require a restart.

This would also help people who add extensions directly into the extensions directory via a separate Dockerfile.